Sources/StreamChat/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -594,8 +594,8 @@ class ChannelController_Tests: StressTestCase {
         }
         XCTAssertNil(error)
         let channel: ChatChannel = client.databaseContainer.viewContext.channel(cid: channelId)!.asModel()
-        assert(channel.latestMessages.count == 1)
-        let message: ChatMessage = channel.latestMessages.first!
+        XCTAssertEqual(channel.latestMessages.count, 1)
+        let message: ChatMessage = try XCTUnwrap(channel.latestMessages.first)
```

Comment:
The semantic difference between `XCTAssert` and `assert` is that by using `assert` you're saying - this is how it should be and it's not the thing we're testing here. However, I do understand the original implementation wasn't the best because it started crashing once you introduced pinned messages. Thanks for improving 👍

Reply:
Yes that was exactly why I changed this, but I understood why it was there in the first place. But it was kinda annoying to always crash when was trying to debug, so that is why I changed 👍